usinggithub.html

```diff
@@ -0,0 +1,118 @@
+<!--
+  CAUTION: This is auto-generated; edit the .src instead.
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>FAQ</title>
+    <link rel="stylesheet" href="css/wgio.min.css">
+    <link rel="icon" type="image/x-icon" href="//labs.w3.org/favicon.ico">
+  </head>
+  <body>
+    <header>
+      <h1>Using Github</h1>
+    </header>
+<nav>
+  <a href="index.html">Home</a>
+  •
+  <a href="https://github.com/w3c/">Repositories</a>
+  •
+  <a href="https://help.github.com/">GitHub Help</a>
+</nav>
+
+<main>
+    <section>
+	  <h2>Why Use Github?</h2>
+	  <p>
+	    W3C groups have been moving to use <a href="https://github.com/">GitHub</a> for reasons including:
+	  </p>
+	    <ul>
+		    <li><strong>Version Control:</strong> Github uses <a href="https://git-scm.com/">Git</a> for version control. This means group members, editors, chairs and the public can easily track and find changes to specifications and other work, refer and rollback to these edits when necessary</li>
+		    <li><strong>User Experience:</strong> Github provides a clean UI for version control and issue management, making it easy for members to use and follow</li>
+		    <li><strong >Issues:</strong> Every Github repository has an issues section. Members can open issues, discuss them, assign them to individuals and label issues for projects and topics. <a href="https://github.com/w3c/payment-request/issues">PaymentRequest</a> gives a good example of well managed Github issues.</li>
+		    <li><strong>Notifications:</strong> Github can send notifications when issues have been discussed, pull requests have been accepted and other general actions have been completed. Guidelines on how to change your notifcation settings can be found <a href="https://help.github.com/articles/choosing-the-types-of-notifications-you-receive/">here</a>.</li>
+		    <li><strong>Integration with W3C Systems:</strong> Some groups and members like to receive their Github notifications via W3C mailing lists. See guidelines on how to do this on the <a href="faq.html">faq</a> page.</li>	    
+		    <li><strong>Global Access:</strong> Github is available around the world and in 日本語 (Japanese), Français (French), Српски (Serbian), Deutsch (German), Svenska (Swedish), Hrvatski (Croatian), Polski (Polish) and Nederlands (Dutch). Github is currently is not blocked in any country, guidelines for country-based and corporate blocks can be found in the <a href="faq.html">faq</a> page. 
```

Comment:
* Are we sure GitHub is available in all those languages? How does one switch language in the platform?
* It seems to me [GitHub Japan](https://github.com/blog/2017-announcing-github-japan) is a separate universe, a different instance (not a localisation of the same platform). eg, `https://github.co.jp/w3c/` does not exist. It may be the case for the other languages, too&hellip;
* These claims (what languages are supported, where it is blocked) are contingent, and we would have to update this paragraph every time something relevant happens&hellip; Is it worth it? I don't see much value.